tests/src/Loader/classloader/generics/Constraints/ConstraintsOnMethod/Negative/app.config

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
is this file still required, i though all dependencies are pulled in via project.json ?